package.json

```diff
@@ -19,7 +19,7 @@
   "scripts": {
     "start": "NODE_ENV=development babel -w ./src --out-dir . --extensions \".ts,.tsx\"",
     "build": "NODE_ENV=production npm run clean && babel ./src --out-dir . --extensions \".ts,.tsx\" && tsc",
-    "clean": "rimraf '+(*.d.ts|*.js)'",
+    "clean": "rimraf *.d.ts *.js",
```

Comment:
Hmm I just double checked `+(*.d.ts|*.js)` locally with:

```bash
npx cli-glob '+(*.d.ts|*.js)'
```

And it didn't match anything in `node_modules`, but separating the globs is simpler.

Reply:
I haven't verified the paths it globs. But somehow my TypeScript compiler compains missing `d.ts` files in `node_modules`.